src/util/profile/profile.ts

```diff
@@ -100,14 +101,15 @@ class ProfileImpl implements Profile {
   }
 }
 
-const validApp = /^([a-zA-Z0-9-_]{3,100})\/([a-zA-Z0-9-_]{3,100})$/;
+const validApp = /^([a-zA-Z0-9-_.]{1,100})\/([a-zA-Z0-9-_.]{1,100})$/;
```

Comment:
I got this regex from Sven and it matched what was on the server. Has this changed on the server?